flytekit/core/python_auto_container.py

```diff
@@ -114,19 +115,25 @@ def get_default_command(self, settings: SerializationSettings) -> List[str]:
         """
         Returns the default pyflyte-execute command used to run this on hosted Flyte platforms.
         """
-        container_args = [
-            "pyflyte-execute",
-            "--inputs",
-            "{{.input}}",
-            "--output-prefix",
-            "{{.outputPrefix}}",
-            "--raw-output-data-prefix",
-            "{{.rawOutputDataPrefix}}",
-            "--resolver",
-            self.task_resolver.location,
-            "--",
-            *self.task_resolver.loader_args(settings, self),
-        ]
+        container_args: List[str] = []
```

Comment:
why do you need this? you can just activate the virtualenv in the docker container.

https://github.com/flyteorg/flytesnacks/blob/master/cookbook/core/Dockerfile#L16-L19

Reply:
We also do something else (statsd setup) in our service_venv

Reply:
@milton0825 then lets add it to the SerializationSettings. I am trying to get rid of the global config access